src/java.base/share/classes/java/util/zip/ZipInputStream.java

```diff
@@ -172,7 +173,7 @@ public void closeEntry() throws IOException {
     }
 
     /**
-     * Returns 0 after EOF has reached for the current entry data,
+     * Returns 0 after EOF has reached for the current ZIP entry,
      * otherwise always return 1.
```

Comment:
I think it will need to say that input stream for the current ZIP entry has been read to end of stream, or the ZIP entry has been closed with closeEntry. That will cover the two cases where it returns 0.

Reply:
> I think it will need to say that input stream for the current ZIP entry has been read to end of stream, or the ZIP entry has been closed with closeEntry. That will cover the two cases where it returns 0.

Thank you Alan.  I have revised the wording per your suggestion and updated the CSR (which would be great if you could review if you are good with the revision.